### DISCUSSION TITLE
Cant seem to override the "cluster-name-kafka-bootstrap" service

### DISCUSSION THREAD

**Discussion by itaydj** [strimzi/strimzi-kafka-operator#4836] [Q&A] [answered]
Hi,
I am creating a procedure for us to migrate to Strimzi based Kafka.
Today, all of our Microservices are connecting to Kafka using a Nodeport service named "Kafka" that points to our old Kafka cluster.
I want all of our services to start using Strimzi Kafka and for that I would like to rename the current "cluster-name-kafka-bootstrap"
to "kafka"
Any idea how to do it?
I went over the documentation and thought I nailed it by adding this to our Kafkacluster yaml:

`
...

template:
    bootstrapService:
      strimzi.io/name: "kafka"

...
`

but it made no difference, Also, I than found out the no strimzi.io label can be changed.

does anyone know if this is possible?
Thanks in advance.

**Comment by scholzj** [comment, marked as answer]
You cannot rename the service. The only thing you can do is to create your own service with your own name but use the selector to point it to the Strimzi pods + if you use TLS add it to the alternative names to make sure it is covered in the certificates for hostname verification. It is not ideal for a permanent solution I guess, but it can give you more time to migrate the clients to the Strimzi service name.

**Reply by itaydj** [reply]
Thanks alot for the quick response.
do you happen to know if this is planned to be supported in the future?

**Reply by scholzj** [reply]
No, I do not think we plan to support this.

**Reply by itaydj** [reply]
I understand, Thanks again Jakub
